boards/khadas/edge2/doc/index.rst

```diff
@@ -0,0 +1,94 @@
+.. _khadas_edge2:
+
+Khadas Edge2
+#################################
+
+Overview
+********
+
+See <https://www.khadas.com/edge2>
+
+Hardware
+********
+
+See <https://docs.khadas.com/products/sbc/edge2/hardware/start>
```

Comment:
please check rendered output https://builds.zephyrproject.io/zephyr/pr/79559/docs/boards/khadas/edge2/doc/index.html and use the correct ReStructuredText way of adding links